### PR TITLE
[FA-13]Patient profile dashboard

### PR DESCRIPTION
**Pull request by johnvamp**
patient profile dasboard

### CODE REVIEW

File: src/components/patientProfile/PatientRecord.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+const PatientRecord = () => {
+  let onClick = () => {
+    return alert("entra");
+  };
+  const citas = ["cita uno", "cita dos", "cita tres"];
```
Comment (SantiguzmaN) [inline src/components/patientProfile/PatientRecord.jsx:7]: please dont use css at jsx files...

File: src/styles/patientProfile.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  border: ridge;
+  border-radius: 100%;
+  background-color: green;
+  margin-top: 20px;
+  margin-left: auto;
+  margin-right: auto;
```
Comment (SantiguzmaN) [inline src/styles/patientProfile.css]: lines 9, 10 and 11 in one line like that...
margin: 20px auto auto auto;

File: src/styles/patientProfile.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+}
+.button-avatar {
+  border-radius: 100%;
+  background-color: gray;
+}
+.button-photo {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 10px;
```
Comment (SantiguzmaN) [inline src/styles/patientProfile.css]: same here with 36, 37, 38

File: src/components/patientProfile/PatientRecord.jsx (hunk 1 above)
Comment (johnvamp) [reply src/components/patientProfile/PatientRecord.jsx:7]: ok

File: src/styles/patientProfile.css (hunk 2 above)
Comment (johnvamp) [reply src/styles/patientProfile.css]: yeah  you're right

**Review by skills20 — APPROVED** [review]

**Review by SantiguzmaN — APPROVED** [review]

**Review by kairox0704 — APPROVED** [review]

**Review by skills20 — APPROVED** [review]

**Pull request merged**